static/js/buddy_data.js

```diff
@@ -179,6 +180,13 @@ export function info_for(user_id) {
 
     const status_emoji_info = user_status.get_status_emoji(user_id);
     const user_circle_status = status_description(user_id);
+    const status_text = user_status.get_status_text(user_id);
+    const right_sidebar_style_value = user_settings.right_sidebar_style;
+    const right_sidebar_style = {
+        COMPACT: right_sidebar_style_value === 1,
+        WITH_STATUS: right_sidebar_style_value === 2,
+        WITH_AVATAR: right_sidebar_style_value === 3,
+    };
```

Comment:
We generally try to put these mappings in `settings_config.js`, rather than where they're used.

Reply:
Any thoughts on how to go about refactoring this? I went with this as there's no straightforward way to compare the setting value inside the hbs file.